index.js

```diff
@@ -12,7 +12,65 @@ function setMultipart (req, done) {
   done()
 }
 
+function attachToBody (options, req, reply, next) {
+  req[kMultipart] = true
+
+  const userWantConsumeStream = typeof options.manageUploadStream === 'function'
+  const consumerStream = options.manageUploadStream
```

Comment:
I think we should implement the accumulating functionality as a default implementation of this function.

Reply:
Yeah, now it is more clear to read